### DISCUSSION TITLE
Hide "Search Indexing was turned off" message

### DISCUSSION THREAD

**Discussion by achtchaern** [krlvm/BeautySearch#35] [Ideas]
Hi there,

if you disable the Search Indexer Service ("WSearch") on Windows 10 22H2, the search app permanently shows a message: "Search Indexing was turned off". The actual search however works flawlessly.

Would it be possible to hide the warning message? It's ugly..

**Comment by krlvm** [comment]
I'll definitely check it

**Comment by achtchaern** [comment]
@krlvm I did some tinkering: After removing some code from the file 15.js, the warning no longer appears! [Code Comparison](https://imgur.com/a/6kMwyM7):

**Comment by krlvm** [comment]
Modifying the original code is not a good solution when it is possible to hide it by adding some CSS

**Reply by achtchaern** [reply]
I understand that, but without any knowledge of these languages, I'm not able to do it the clean way.
